src/framework/shortcuts/view/shortcutsmodel.cpp

```diff
@@ -70,6 +71,17 @@ const QString& ShortcutsModel::actionTitle(const std::string& actionCode) const
     return action.title;
 }
 
+const QString& ShortcutsModel::actionDescription(const std::string& actionCode) const
+{
+    const UiAction& action = this->action(actionCode);
+    return action.description;
+}
+
+const QString& ShortcutsModel::actionText(const std::string& actionCode) const
+{
+    return actionDescription(actionCode) == "" ? actionTitle(actionCode) : actionDescription(actionCode);
```

Comment:
It looks like `actionDescription(actionCode) == ""` will never be true, because in `load()`, you do:
```c++
         if (action.title.isEmpty() || action.description.isEmpty()) {
             continue;
         }
```
Maybe you meant `&&` instead of `||`? Or is the emptiness check just redundant?

Reply:
I agree that this should never really be required. But just as a safety measure, if somehow a description was empty and it was allowed in the list(maybe inserted via some other measure and modified later on), it should then default to the title.

Reply:
@VanSHOE 
Please optimize the code, you are calculating the description twice.